packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1932,37 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Storybook',
+    slug: 'storybook',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/storybook.svg',
+    tagline: 'Frontend workshop for UI development',
+    description:
+      'Storybook is a frontend workshop for building UI components and pages in isolation.',
+    website: 'https://storybook.js.org',
+    ignoreRuntimes: ['@vercel/next', '@vercel/node'],
+    detectors: {
+      every: [
+        {
+          matchPackage: 'storybook',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install`, `pnpm install`, or `npm install`',
+      },
+      buildCommand: {
+        value: 'storybook build',
+      },
+      devCommand: {
+        value: `start-storybook -p $PORT`,
```

Comment:
```suggestion
        value: `storybook dev -p $PORT`,
```

Where did you get `start-storybook` from? 

The `examples/storybook/package.json` seems to indicate this is `storybook dev`

Reply:
Good question. Copy and paste error.